config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -15,6 +15,23 @@ presets:
 
 presubmits:
   kubernetes/minikube:
+  - name: pull-minikube-build
+    decorate: true
+    path_alias: "k8s.io/minikube"
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: docker.io/azhao155/prow-test:1.7
```

Comment:
Please don't use a dockerhub-hosted image, or an image from a personal project/repo.  Whatever was used to produce this image needs to be open-source and reproducible by others.

What needs to be in this image, and does one of the existing images in gcr.io/k8s-testimages satisfy your needs?  If not, what is missing?

Reply:
thats a good idea ! thanks @spiffxp I will provide access to @azhao155 to push the image to k8s-minikube project

Reply:
Updated image and add OWNER file.